extensions/libarchive/MergeContent.cpp

```diff
@@ -72,10 +74,12 @@ core::Property MergeContent::AttributeStrategy(
                     "only the attributes that exist on all FlowFiles in the bundle, with the same value, will be preserved.")
   ->withAllowableValues<std::string>({merge_content_options::ATTRIBUTE_STRATEGY_KEEP_COMMON, merge_content_options::ATTRIBUTE_STRATEGY_KEEP_ALL_UNIQUE})
   ->withDefaultValue(merge_content_options::ATTRIBUTE_STRATEGY_KEEP_COMMON)->build());
+core::Property MergeContent::FlowFileSerializer(
+  core::PropertyBuilder::createProperty("Flow File Serializer")
+  ->withDescription("Determines how to flow files should be serialized before merging")
+  ->withAllowableValues<std::string>({merge_content_options::SERIALIZER_PAYLOAD, merge_content_options::SERIALIZER_FLOW_FILE_V3})
+  ->withDefaultValue(merge_content_options::SERIALIZER_PAYLOAD)->build());
```

Comment:
NiFi uses the merge format property for this purpose, with the new serializer bound to the equivalent of `BinaryConcatenationMerge`. Why is it separate here?

Reply:
imo the idea of WHAT we want to merge, and HOW we want to merge are orthogonal, here `Flow File Serializer` answers the WHAT and `Merge Format` answers the HOW

Reply:
I agree on the implementation level, but on the interface, compatibility with NiFi offers a greater benefit than separation of concerns IMO. Now you can create a tar of FFv3-serialized flow files, but what would you do with it if NiFi can't handle it?

Reply:
should be a straightforward change, this being an interface level decision I would also like to ask @arpadboda to confirm dropping the separate `Flow File Serializer` property and merging it into `Merge Format`

Reply:
The current implementation makes sense in a way, but I also think that compatibility with NiFi would bring more benefits, so please merge in case it's not a big change.

Reply:
the serializer has been integrated into the Merge Format (using the NiFi-equivalent `FlowFile Stream, v3` name)